src/System.IO.Pipelines/src/Configurations.props

```diff
@@ -3,6 +3,7 @@
     <PackageConfigurations>
       netstandard;
       netcoreapp;
+      netcoreapp3.0;
```

Comment:
Can you please explain how you came up with the list of additional build configurations?

Reply:
When you add a netcoreapp3.0 configuration to the `PackageConfigurations`, you should move the 'netcoreapp' configuration to a `BuildConfiguration`

Reply:
Wrote a little bit of code to get the list 
```C#
List<string> paths = Directory.GetFiles(@"C:\git\corefx\src", "Configurations.props", SearchOption.AllDirectories).ToList();
paths = paths.Where(t => !t.Contains(@"\tests\")).ToList();

foreach (var path in paths)
{
    Project t = new Project(path);
    foreach (var item in t.AllEvaluatedProperties)
    {
        if (item.Name == "PackageConfigurations")
        {
string k = item.EvaluatedValue.Trim(); 
if (k.Contains("netcoreapp;") || k.Contains("netcoreapp-Windows_NT") || k.Contains("netcoreapp-Unix_NT"))
{
    Console.WriteLine(path);
}
        }
    }
}
```

It gave me an output as 
```
C:\git\corefx\src\Microsoft.Win32.SystemEvents\src\Configurations.props
C:\git\corefx\src\System.IO.Pipelines\src\Configurations.props
C:\git\corefx\src\System.Runtime.WindowsRuntime\src\Configurations.props
C:\git\corefx\src\System.Runtime.WindowsRuntime.UI.Xaml\src\Configurations.props
C:\git\corefx\src\System.Security.Cryptography.Cng\ref\Configurations.props
C:\git\corefx\src\System.Security.Cryptography.Cng\src\Configurations.props
C:\git\corefx\src\System.Security.Cryptography.Pkcs\ref\Configurations.props
C:\git\corefx\src\System.Security.Cryptography.Pkcs\src\Configurations.props
C:\git\corefx\src\System.Security.Permissions\ref\Configurations.props
C:\git\corefx\src\System.Security.Permissions\src\Configurations.props
C:\git\corefx\src\System.Security.Principal.Windows\ref\Configurations.props
C:\git\corefx\src\System.Text.Json\src\Configurations.props
C:\git\corefx\src\System.Threading.Channels\src\Configurations.props
```